tests/virt_autotest/virt_utils.pm

```diff
@@ -545,4 +545,14 @@ sub generate_testcase_xml {
     $xml_writer->endTag('testcase');
 }
 
+#RAW do not support Snapshot, so skip Snapshot test if guest disk type as RAW
+sub check_guest_disk_type {
+    my $guest           = shift;
+    my $guest_disk_type = script_output("virsh dumpxml $guest | grep \"<driver \" | grep -o \"type='.*'\" | cut -d \"'\" -f2");
+    if ($guest_disk_type =~ /raw/) {
```

Comment:
This function seems to need to return 1 for non-raw and 0 for raw disk type. Currently it can not differ raw and non-raw for further action.

Reply:
Had updated code already.